eng/common/pipelines/templates/steps/docindex.yml

```diff
@@ -1,7 +1,5 @@
 jobs:
   - job: CreateDocIndex
-    variables:
```

Comment:
I assume you plan to make this in the tools repo once you get this working here.

Reply:
Right, this is a miss when I do testing. Since the docindex is not in master, I haven't done full testing on pipeline. Will test all repo and submit another change in tool repo.